Cargo.toml

```diff
@@ -30,15 +30,23 @@ void = { version = "1.0.2", default-features = false }
 cast = { version = "0.2.3", default-features = false }
 nb = "0.1.2"
 paste = "0.1.18"
+log = "0.4.11"
+panic-halt = "0.2.0"
```

Comment:
Does panic-halt really have to be in our default enabled dependencies? I personally don't like the idea of having a default panic implementation as a HAL when usually the user wants to determine that implementation in their crate themselves.

Reply:
Agreed, having panic implementation as a required dependency is not helpful